src/it/resources/org/checkstyle/suppressionxpathfilter/upperell/SuppressionXpathRegressionUpperEllDefault.java

```diff
@@ -0,0 +1,11 @@
+/*
+UpperEll
+
+*/
+
+package org.checkstyle.suppressionxpathfilter.upperell;
+
+public class SuppressionXpathRegressionUpperEllDefault {
+    long var1 = 508987;
+    long var2 = 508987l; //warn
+}
```

Comment:
This is almost the same as the first input code. Please make the code structure different from the first test: replace `class` with `interface`, define `long` variable in a method or anonymouse class, etc...

Reply:
Changes Done